docs/markdown/Reference-manual.md

```diff
@@ -1398,13 +1398,14 @@ Project supports the following keyword arguments.
   specific options are used normally even in subprojects.
 
 
-- `license`: takes a string or array of strings describing the
-  license(s) the code is under. Usually this would be something like
-  `license : 'GPL2+'`, but if the code has multiple licenses you can
-  specify them as an array like this: `license : ['proprietary',
-  'GPL3']`. Note that the text is informal and is only written to
-  the dependency manifest. Meson does not do any license validation,
-  you are responsible for verifying that you abide by all licensing
+- `license`: takes a string or array of strings describing the license(s) the
+  code is under. To avoid ambiguity it is recommended to use a standardized
+  license identifier from the [SPDX license list](https://spdx.org/licenses/).
+  Usually this would be something like `license : 'GPL-2.0-or-later'`, but if
+  the code has multiple licenses you can specify them as an array like this:
```

Comment:
As you mention, the array is ambiguous as well so I would first recommend `AND` and `OR` connectives in this sentence. The array syntax can be described with the description that it  is actually informal. Eventually, we might want to deprecate the informal license.

Reply:
I'm :+1: on that but it's a more minor thing, linking to https://spdx.org/licenses/ already partly covers it (at least if one looks more into the SPDX standard), and I'm not sure if the consensus would be to deprecate the array (as it would obviously be a breaking change for some). Feel free to open another PR/issue for that or briefly continue the discussion in the already closed issue (if the maintainers don't mind; in this case it'd seem ideal for a quick feedback if/what we want to do about this).